api/lib/database/helpers.js

```diff
@@ -133,7 +133,11 @@ const insert = ({ hasura_url, hasura_admin_key }, upsert = false) => (
             }
     }
     `
-  return graphQLQuery({ hasura_url, hasura_admin_key })(query)
+  const response = await graphQLQuery({ hasura_url, hasura_admin_key })(query)
+  // return the inserted object
+  if (returnKeys.length > 0) {
+    return response.data[`insert_${schema_prefix}${table}`].returning
+  }
```

Comment:
what is this? I had never seen that `returning` before?

Reply:
this returns the inserted row. I used this to get the created UUID.